resources/css/bem/input-container.less

```diff
@@ -3,20 +3,37 @@
 
 .input-container {
   --label-colour: hsl(var(--hsl-c2));
-  --input-bg: hsl(var(--hsl-b5));
+  --label-padding-base: 5px;
+  --input-bg: hsl(var(--hsl-b6));
+  --input-border-colour: var(--input-border-base-colour);
+  --input-border-base-colour: transparent;
+  --input-border-hover-colour: hsl(var(--hsl-l3));
+  --input-border-focus-colour: hsl(var(--hsl-l1));
+  --input-border-error-colour: hsl(var(--hsl-red-2));
+  --input-padding: var(--label-padding-base) var(--input-padding-base)
```

Comment:
What's the reason for splitting up the container's inner padding into label and input outer padding (aka margin)?

Reply:
the end result would be to absolute position the label and have the input take the whole area. It's mainly for select input so clicking anywhere will show the dropdown